sdk/storage/azblob/internal/generated/autorest.md

```diff
@@ -7,7 +7,7 @@ go: true
 clear-output-folder: false
 version: "^3.0.0"
 license-header: MICROSOFT_MIT_NO_VERSION
-input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/080b332b7572514a2e100dd2fa1fb86cb8edcb08/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json"
+input-file: "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a32d0b2423d19835246bb2ef92941503bfd5e734/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json"
```

Comment:
STG 87 is `2022-11-02`. But the above URL is pointing to `2021-12-02`.

Reply:
We don't have a swagger file for blobs beyond 2021-12-02 yet, we only add new files where there is a feature for the service in question that requires a swagger change and a new version.

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/storage/data-plane/Microsoft.BlobStorage/preview